pkg/minikube/assets/addons.go

```diff
@@ -75,7 +75,7 @@ var Addons = map[string]*Addon{
 			"addon-manager.yaml.tmpl",
 			"0640",
 			true),
-	}, true, "addon-manager"),
+	}, false, "addon-manager"),
```

Comment:
Please follow this up with a PR that removes the addon-manager. I think it's continued existence will only add confusion. 

If someone begs for the addon-manager to  come back, we can reconsider.

Reply:
sounds good